plugins/ccgx/fu-ccgx-hpi-device.c

```diff
@@ -1379,9 +1376,11 @@ fu_ccgx_hpi_device_setup (FuDevice *device, GError **error)
 		fu_device_add_flag (FU_DEVICE (self), FWUPD_DEVICE_FLAG_UPDATABLE);
 	}
 
-	/* set name to be more descriptive */
-	name = g_strdup_printf ("USB-I2C Bridge (%s)", fu_ccgx_hpi_device_get_name (self));
-	fu_device_set_name (FU_DEVICE (self), name);
+	/* set default device name */
+	if (fu_device_get_name(FU_DEVICE (self)) == NULL) 
+		fu_device_set_name (FU_DEVICE (self), "USB-I2C Bridge");		
```

Comment:
Do we need these last two lines? The name should be set from the USB descriptor by default.

Reply:
I fixed it. 
Please check [https://github.com/fwupd/fwupd/pull/2013/commits/605b1fce02e8d9e66e7e31c790f0d82ce6635827](url)